src/main/resources/liquibase/workbench_changelog/14_4_0.xml

```diff
@@ -36,4 +36,23 @@
 		</sql>
 	</changeSet>
 
+	<changeSet author="cuenyad" id="v14.4.0-2">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM permission where name = 'CANCEL_PENDING_TRANSACTIONS'
+            </sqlCheck>
+        </preConditions>
+        <comment>DEPOSIT_INVENTORY granular permissions</comment>
+		<sql dbms="mysql" splitStatements="true">
+			set @manage_lots_id = (select permission_id from permission where name = 'MANAGE_TRANSACTIONS');
+
+			insert into permission(name, description, parent_id, workbench_sidebar_category_link_id)
+			values ('CANCEL_PENDING_TRANSACTIONS', 'Cancel pending transactions', @manage_lots_id, null);
+			set @deposit_inventory_id = (select last_insert_id());
+			set @last_permission_id  = (select last_insert_id());
+			insert into role_type_permission(role_type_id, permission_id, selectable) values (1, @last_permission_id , 1);
+			insert into role_type_permission(role_type_id, permission_id, selectable) values (2, @last_permission_id , 1);
+		</sql>
+	</changeSet>
+
```

Comment:
naming changes, the changeset seems ok
`<comment>DEPOSIT_INVENTORY granular permissions</comment>` -> _Cancel pending transactions permissions_ 
`@manage_lots_id` to `@manage_transactions_id`